pkg/chunk/object/chunk_object_client.go

```diff
@@ -0,0 +1,80 @@
+package object
```

Comment:
A couple of things:
1. The filename doesn't look reflecting the `type` name
2. Is it required adding a new pkg just for this file? I'm wondering if we could move it to `pkg/chunk/` and rename this `ChunkClient` to something like `ChunkStoreClient` to avoid conflicts

Reply:
This can’t live in the chunk package because of a circular dependency issue with the util package.

Reply:
I didn't notice it. Then ok.

> The filename doesn't look reflecting the type name

I believe this can still be renamed to reflect the struct name.

Reply:
I agree I’m struggling to come up with a name that’s not to verbose but denotes a chunk client abstraction on top of an object store

Reply:
I decided to call the package `objectclient` and the struct `Client`. That way it will be the `chunk/objectclient.Client` struct. This seems conceptually sound to me. Let me know what you think.